lib/msal-common/src/error/InteractionRequiredAuthError.ts

```diff
@@ -40,12 +41,30 @@ export const InteractionRequiredAuthErrorMessage = {
  * Error thrown when user interaction is required.
  */
 export class InteractionRequiredAuthError extends AuthError {
+    /**
+     * The time the error occured at
+     */
+    timestamp: string;
 
-    constructor(errorCode?: string, errorMessage?: string, subError?: string) {
-        super(errorCode, errorMessage, subError);
-        this.name = "InteractionRequiredAuthError";
+    /**
+     * TraceId associated with the error
+     */
+    traceId: string;
 
+    /**
+     * Claims associated with the error that can be used to get a token
```

Comment:
Please add a more descriptive exception of what the developer should do with claims, somethinlike 

`A string with extra claims needed for the token request to succeed. If in a web site, redirect the user to the authorization page and set the extra claims. If in a web api, make sure to include the claims in the WWW-Authenticate header that you send to the client, so that the client knows to request a token with the extra claims. If in a desktop application or in a browser context, include the claims when acquiring the token interactively. `

CC @derisen for the comment.

Reply:
@bgavrilMS Is there documentation where I can find more info on this? Where did you pull all those examples from?

Reply:
Taking another look at that doc, it needs to be a little more generalized to cover CA properly (challenge coming from STS vs resource server etc.). I made a note of that and will update, but we can still place the link here I think.